mock/data/restructuredData.json.txt

```diff
@@ -0,0 +1,2 @@
+nodes with match in link:
```

Comment:
Still required, you found the bug, right?

Reply:
Correct. Not required anymore.